raw_install_python.yml

```diff
@@ -2,26 +2,44 @@
 - name: check for python
   stat:
     path: /usr/bin/python
-  ignore_errors: yes
+  changed_when: false
+  failed_when: false
   register: systempython
 
-- name: install python for debian based systems
-  raw: apt-get -y install python-simplejson
-  ignore_errors: yes
-  register: result
-  when: systempython.stat is undefined or not systempython.stat.exists
-  until: result is succeeded
+- block:
+    - name: check for dnf-3 package manager (RedHat/Fedora/CentOS)
+      raw: stat /bin/dnf-3
+      changed_when: false
+      failed_when: false
+      register: stat_dnf3
 
-- name: install python for fedora
-  raw: dnf -y install python3; ln -sf /usr/bin/python3 /usr/bin/python creates=/usr/bin/python
-  ignore_errors: yes
-  register: result
-  when: systempython.stat is undefined or not systempython.stat.exists
-  until: (result is succeeded) and ('Failed' not in result.stdout)
+    - name: check for apt package manager (Debian/Ubuntu)
+      raw: stat /bin/apt
```

Comment:
fix the typo s/apt/apt-get` + need to double check whether it's well located in `/bin`